schedule/yast/maintenance/autoupgrade_sles12sp5_to_sles15spx_maintenance.yaml

```diff
@@ -0,0 +1,28 @@
+---
+name: autoupgrade
```

Comment:
It's better change the name same with files' name.

Reply:
updated, thanks.